scripts/deploy.sh

```diff
@@ -33,12 +33,14 @@ function upload_assets {
   mv -f "${TLDR_LANG_ARCHIVES_DIRECTORY}"/*.zip "$SITE_HOME/assets/"
   rm -rf "$TLDR_LANG_ARCHIVES_DIRECTORY"
   cp -f "$TLDRHOME/index.json" "$SITE_HOME/assets/"
-  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages.pdf" "${SITE_HOME}/assets/tldr-book.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages*.pdf" "${SITE_HOME}/assets/"
```

Comment:
Oh, this is actually a breaking change. Is there any reason to have the intermediary PDF be written out as `tldr-pages.pdf` instead of `tldr-book.pdf`? Or is it fine to have it be `tldr-pages.pdf`?

Could also just make a copy of `tldr-pages.pdf` to `tldr-book.pdf` as well as keep all of `tldr-pages*.pdf` with `tldr-book.pdf` only existing as a historical artifact and that everyone should instead move to `tldr-pages.pdf` (or the relevant language PDF they want).